tests/installation/addon_products_sle.pm

```diff
@@ -178,17 +179,13 @@ sub run {
             if (match_has_tag("addon-betawarning-$addon") or match_has_tag("addon-license-$addon")) {
                 if (match_has_tag("addon-betawarning-$addon")) {
                     send_key "ret";
-                    assert_screen [qw(addon-license-beta addon-products)];
-                    record_soft_failure 'bsc#1057223: No license agreement shown when HA, HA-GEO, WE, RT extensions are added as addons'
-                      unless match_has_tag("addon-license-beta");
-                }
-                if (match_has_tag("addon-license-beta") or match_has_tag("addon-license-$addon")) {
-                    wait_still_screen 2;
-                    send_key 'alt-a';                     # yes, agree
-                    wait_still_screen 2;
-                    send_key $cmd{next};
-                    assert_screen 'addon-products', 90;
+                    assert_screen "addon-license-beta";
```

Comment:
I'm not sure if this is safe change, as we have no-beta licenses too. So I guess we have to assert on both tags as before. And I'm not sure if we break something if we start to accept license unconditionally. If we bring array ref here it should be fine, but I would like to see some verification runs, especially for migration jobs.

Reply:
I did not change that part of code. I just reverted it to the previous state.